.azuredevops/platformPipelines/platform.dependencies.yml

```diff
@@ -25,7 +25,7 @@ trigger: none
 #       - 'utilities/pipelines/dependencies/**'
 
 variables:
-  - template: '../../global.variables.yml'
```

Comment:
Did you run the pipeline end to end?

Reply:
yes. But need to run another set of tests after some major PRs being merged. Will update status